kafka_utils/kafka_consumer_manager/commands/offset_manager.py

```diff
@@ -144,14 +144,23 @@ def get_topics_for_group_from_kafka(
         groupid,
     ):
         kafka_group_reader = KafkaGroupReader(cluster_config)
-        group_topics = kafka_group_reader.read_group(groupid)
-        if len(group_topics) == 0:
+        try:
+            group_topics = kafka_group_reader.read_group(groupid)
+        except KeyError:
             print(
-                "Error: Either Consumer Group {groupid} has never committed offsets, or the group does not exist.".format(
+                "Error: Consumer Group {groupid} does not exist.".format(
```

Comment:
What about changing it to something like "Warning: Consumer Group X does not exist. Creating it."? It is not really an error and we just go forward anyway.

Reply:
This function is called in other situations as well. I think that's why I made the print message intentionally vague at the time :/